acos_client/v30/slb/hm.py

```diff
@@ -93,7 +93,6 @@ def _set(self, name, mon_method, hm_interval, hm_timeout, hm_max_retries,
             else:
                 k = '%s-port' % mon_method
             params['monitor']['method'][mon_method][k] = int(port)
-            params['monitor']['override-port'] = int(port)
```

Comment:
Please provide an option for other user that still able to use the override-port